pycg/processing/postprocessor.py

```diff
@@ -128,16 +142,6 @@ def visit_For(self, node):
         super().visit_For(node)
         logger.debug("Exit PreProcessor.visit_For")
 
-    def visit_Return(self, node):
```

Comment:
Whole methods were duplicated, redefining the same name? I believe there's one more method redefinition in the project, but the two implementations are not the same, so I've left those be for now.